Not compatible with Node.JS v11.0.0

```
> font-manager@0.2.2 install /home/spaceboyross/RDE/src/panel/node_modules/font-manager
> node-gyp rebuild

make: Entering directory '/home/spaceboyross/RDE/src/panel/node_modules/font-manager/build'
  CXX(target) Release/obj.target/fontmanager/src/FontManager.o
In file included from ../node_modules/nan/nan.h:194,
                 from ../src/FontManager.cc:5:
../node_modules/nan/nan_maybe_43_inl.h: In function ‘Nan::Maybe<bool> Nan::ForceSet(v8::Local<v8::Object>, v8::Local<v8::Value>, v8::Local<v8::Value>, v8::PropertyAttribute)’:
../node_modules/nan/nan_maybe_43_inl.h:88:15: error: ‘class v8::Object’ has no member named ‘ForceSet’
   return obj->ForceSet(GetCurrentContext(), key, value, attribs);
               ^~~~~~~~
../node_modules/nan/nan_maybe_43_inl.h: In function ‘Nan::MaybeLocal<v8::Object> Nan::CloneElementAt(v8::Local<v8::Array>, uint32_t)’:
../node_modules/nan/nan_maybe_43_inl.h:220:17: error: ‘class v8::Array’ has no member named ‘CloneElementAt’
   return array->CloneElementAt(GetCurrentContext(), index);
                 ^~~~~~~~~~~~~~
In file included from ../node_modules/nan/nan_new.h:189,
                 from ../node_modules/nan/nan.h:200,
                 from ../src/FontManager.cc:5:
../node_modules/nan/nan_implementation_12_inl.h: In static member function ‘static Nan::imp::FactoryBase<v8::BooleanObject>::return_t Nan::imp::Factory<v8::BooleanObject>::New(bool)’:
../node_modules/nan/nan_implementation_12_inl.h:40:38: error: no matching function for call to ‘v8::BooleanObject::New(bool&)’
   return v8::BooleanObject::New(value).As<v8::BooleanObject>();
                                      ^
In file included from /home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:63,
                 from ../src/FontManager.cc:2:
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h:5218:23: note: candidate: ‘static v8::Local<v8::Value> v8::BooleanObject::New(v8::Isolate*, bool)’
   static Local<Value> New(Isolate* isolate, bool value);
                       ^~~
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h:5218:23: note:   candidate expects 2 arguments, 1 provided
In file included from ../node_modules/nan/nan_new.h:189,
                 from ../node_modules/nan/nan.h:200,
                 from ../src/FontManager.cc:5:
../node_modules/nan/nan_implementation_12_inl.h:40:60: error: expected primary-expression before ‘>’ token
   return v8::BooleanObject::New(value).As<v8::BooleanObject>();
                                                            ^
../node_modules/nan/nan_implementation_12_inl.h:40:62: error: expected primary-expression before ‘)’ token
   return v8::BooleanObject::New(value).As<v8::BooleanObject>();
                                                              ^
../node_modules/nan/nan_implementation_12_inl.h: In static member function ‘static Nan::imp::FactoryBase<v8::StringObject>::return_t Nan::imp::Factory<v8::StringObject>::New(v8::Local<v8::String>)’:
../node_modules/nan/nan_implementation_12_inl.h:340:37: warning: ‘static v8::Local<v8::Value> v8::StringObject::New(v8::Local<v8::String>)’ is deprecated: Use Isolate* version [-Wdeprecated-declarations]
   return v8::StringObject::New(value).As<v8::StringObject>();
                                     ^
In file included from /home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h:26,
                 from /home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:63,
                 from ../src/FontManager.cc:2:
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h:5236:37: note: declared here
                 static Local<Value> New(Local<String> value));
                                     ^~~
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8config.h:326:3: note: in definition of macro ‘V8_DEPRECATED’
   declarator __attribute__((deprecated(message)))
   ^~~~~~~~~~
In file included from ../node_modules/nan/nan_new.h:189,
                 from ../node_modules/nan/nan.h:200,
                 from ../src/FontManager.cc:5:
../node_modules/nan/nan_implementation_12_inl.h:340:37: warning: ‘static v8::Local<v8::Value> v8::StringObject::New(v8::Local<v8::String>)’ is deprecated: Use Isolate* version [-Wdeprecated-declarations]
   return v8::StringObject::New(value).As<v8::StringObject>();
                                     ^
In file included from /home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h:26,
                 from /home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:63,
                 from ../src/FontManager.cc:2:
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h:5236:37: note: declared here
                 static Local<Value> New(Local<String> value));
                                     ^~~
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8config.h:326:3: note: in definition of macro ‘V8_DEPRECATED’
   declarator __attribute__((deprecated(message)))
   ^~~~~~~~~~
In file included from ../src/FontManager.cc:5:
../node_modules/nan/nan.h: In function ‘v8::Local<v8::Value> Nan::MakeCallback(v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)’:
../node_modules/nan/nan.h:821:60: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
         v8::Isolate::GetCurrent(), target, func, argc, argv);
                                                            ^
In file included from ../src/FontManager.cc:2:
/home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:171:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^~~~~~~~~~~~
/home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:91:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/FontManager.cc:5:
../node_modules/nan/nan.h:821:60: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
         v8::Isolate::GetCurrent(), target, func, argc, argv);
                                                            ^
In file included from ../src/FontManager.cc:2:
/home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:171:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^~~~~~~~~~~~
/home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:91:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/FontManager.cc:5:
../node_modules/nan/nan.h: In function ‘v8::Local<v8::Value> Nan::MakeCallback(v8::Local<v8::Object>, v8::Local<v8::String>, int, v8::Local<v8::Value>*)’:
../node_modules/nan/nan.h:835:62: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::String>, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
         v8::Isolate::GetCurrent(), target, symbol, argc, argv);
                                                              ^
In file included from ../src/FontManager.cc:2:
/home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:164:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^~~~~~~~~~~~
/home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:91:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/FontManager.cc:5:
../node_modules/nan/nan.h:835:62: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::String>, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
         v8::Isolate::GetCurrent(), target, symbol, argc, argv);
                                                              ^
In file included from ../src/FontManager.cc:2:
/home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:164:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^~~~~~~~~~~~
/home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:91:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/FontManager.cc:5:
../node_modules/nan/nan.h: In function ‘v8::Local<v8::Value> Nan::MakeCallback(v8::Local<v8::Object>, const char*, int, v8::Local<v8::Value>*)’:
../node_modules/nan/nan.h:849:62: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, const char*, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
         v8::Isolate::GetCurrent(), target, method, argc, argv);
                                                              ^
In file included from ../src/FontManager.cc:2:
/home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:157:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^~~~~~~~~~~~
/home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:91:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/FontManager.cc:5:
../node_modules/nan/nan.h:849:62: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, const char*, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
         v8::Isolate::GetCurrent(), target, method, argc, argv);
                                                              ^
In file included from ../src/FontManager.cc:2:
/home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:157:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^~~~~~~~~~~~
/home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:91:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/FontManager.cc:5:
../node_modules/nan/nan.h: In constructor ‘Nan::Utf8String::Utf8String(v8::Local<v8::Value>)’:
../node_modules/nan/nan.h:894:53: warning: ‘v8::Local<v8::String> v8::Value::ToString() const’ is deprecated: Use maybe version [-Wdeprecated-declarations]
       v8::Local<v8::String> string = from->ToString();
                                                     ^
In file included from /home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:63,
                 from ../src/FontManager.cc:2:
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h:10242:15: note: declared here
 Local<String> Value::ToString() const {
               ^~~~~
In file included from ../src/FontManager.cc:5:
../node_modules/nan/nan.h:904:74: warning: ‘int v8::String::WriteUtf8(char*, int, int*, int) const’ is deprecated: Use Isolate* version [-Wdeprecated-declarations]
         length_ = string->WriteUtf8(str_, static_cast<int>(len), 0, flags);
                                                                          ^
In file included from /home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h:26,
                 from /home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:63,
                 from ../src/FontManager.cc:2:
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h:2754:21: note: declared here
                 int WriteUtf8(char* buffer, int length = -1,
                     ^~~~~~~~~
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8config.h:326:3: note: in definition of macro ‘V8_DEPRECATED’
   declarator __attribute__((deprecated(message)))
   ^~~~~~~~~~
In file included from ../src/FontManager.cc:5:
../node_modules/nan/nan.h: In member function ‘v8::Local<v8::Value> Nan::Callback::Call_(v8::Isolate*, v8::Local<v8::Object>, int, v8::Local<v8::Value>*) const’:
../node_modules/nan/nan.h:1467:5: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
     ));
     ^
In file included from ../src/FontManager.cc:2:
/home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:171:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^~~~~~~~~~~~
/home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:91:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/FontManager.cc:5:
../node_modules/nan/nan.h:1467:5: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
     ));
     ^
In file included from ../src/FontManager.cc:2:
/home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:171:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^~~~~~~~~~~~
/home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:91:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/FontManager.cc:5:
../node_modules/nan/nan.h: In function ‘void Nan::AsyncQueueWorker(Nan::AsyncWorker*)’:
../node_modules/nan/nan.h:1707:62: warning: cast between incompatible function types from ‘void (*)(uv_work_t*)’ {aka ‘void (*)(uv_work_s*)’} to ‘uv_after_work_cb’ {aka ‘void (*)(uv_work_s*, int)’} [-Wcast-function-type]
     , reinterpret_cast<uv_after_work_cb>(AsyncExecuteComplete)
                                                              ^
../node_modules/nan/nan.h: In function ‘bool Nan::SetAccessor(v8::Local<v8::Object>, v8::Local<v8::String>, Nan::GetterCallback, Nan::SetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute)’:
../node_modules/nan/nan.h:1961:16: error: no matching function for call to ‘v8::Object::SetAccessor(v8::Local<v8::String>&, void (*&)(v8::Local<v8::Name>, const v8::PropertyCallbackInfo<v8::Value>&), void (*&)(v8::Local<v8::Name>, v8::Local<v8::Value>, const v8::PropertyCallbackInfo<void>&), v8::Local<v8::Object>&, v8::AccessControl&, v8::PropertyAttribute&)’
     , attribute);
                ^
In file included from /home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:63,
                 from ../src/FontManager.cc:2:
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h:3454:37: note: candidate: ‘v8::Maybe<bool> v8::Object::SetAccessor(v8::Local<v8::Context>, v8::Local<v8::Name>, v8::AccessorNameGetterCallback, v8::AccessorNameSetterCallback, v8::MaybeLocal<v8::Value>, v8::AccessControl, v8::PropertyAttribute, v8::SideEffectType)’
   V8_WARN_UNUSED_RESULT Maybe<bool> SetAccessor(
                                     ^~~~~~~~~~~
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h:3454:37: note:   no known conversion for argument 2 from ‘Nan::imp::NativeGetter’ {aka ‘void (*)(v8::Local<v8::Name>, const v8::PropertyCallbackInfo<v8::Value>&)’} to ‘v8::Local<v8::Name>’
In file included from ../src/FontManager.cc:6:
../src/FontDescriptor.h: In member function ‘int FontDescriptor::getNumber(v8::Local<v8::Object>, const char*)’:
../src/FontDescriptor.h:152:32: warning: ‘int32_t v8::Value::Int32Value() const’ is deprecated: Use maybe version [-Wdeprecated-declarations]
       return value->Int32Value();
                                ^
In file included from /home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h:26,
                 from /home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:63,
                 from ../src/FontManager.cc:2:
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h:2572:46: note: declared here
   V8_DEPRECATED("Use maybe version", int32_t Int32Value() const);
                                              ^~~~~~~~~~
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8config.h:326:3: note: in definition of macro ‘V8_DEPRECATED’
   declarator __attribute__((deprecated(message)))
   ^~~~~~~~~~
In file included from ../src/FontManager.cc:6:
../src/FontDescriptor.h: In member function ‘bool FontDescriptor::getBool(v8::Local<v8::Object>, const char*)’:
../src/FontDescriptor.h:163:34: warning: ‘bool v8::Value::BooleanValue() const’ is deprecated: Use maybe version [-Wdeprecated-declarations]
       return value->BooleanValue();
                                  ^
In file included from /home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h:26,
                 from /home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:63,
                 from ../src/FontManager.cc:2:
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h:2568:43: note: declared here
   V8_DEPRECATED("Use maybe version", bool BooleanValue() const);
                                           ^~~~~~~~~~~~
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8config.h:326:3: note: in definition of macro ‘V8_DEPRECATED’
   declarator __attribute__((deprecated(message)))
   ^~~~~~~~~~
In file included from ../src/FontManager.cc:2:
../src/FontManager.cc: At global scope:
/home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:494:43: warning: cast between incompatible function types from ‘void (*)(Nan::ADDON_REGISTER_FUNCTION_ARGS_TYPE)’ {aka ‘void (*)(v8::Local<v8::Object>)’} to ‘node::addon_register_func’ {aka ‘void (*)(v8::Local<v8::Object>, v8::Local<v8::Value>, void*)’} [-Wcast-function-type]
       (node::addon_register_func) (regfunc),                          \
                                           ^
/home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:528:3: note: in expansion of macro ‘NODE_MODULE_X’
   NODE_MODULE_X(modname, regfunc, NULL, 0)  // NOLINT (readability/null_usage)
   ^~~~~~~~~~~~~
../src/FontManager.cc:223:1: note: in expansion of macro ‘NODE_MODULE’
 NODE_MODULE(fontmanager, Init)
 ^~~~~~~~~~~
In file included from /home/spaceboyross/.node-gyp/11.0.0/include/node/node.h:63,
                 from ../src/FontManager.cc:2:
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h: In instantiation of ‘void v8::PersistentBase<T>::SetWeak(P*, typename v8::WeakCallbackInfo<P>::Callback, v8::WeakCallbackType) [with P = node::ObjectWrap; T = v8::Object; typename v8::WeakCallbackInfo<P>::Callback = void (*)(const v8::WeakCallbackInfo<node::ObjectWrap>&)]’:
/home/spaceboyross/.node-gyp/11.0.0/include/node/node_object_wrap.h:85:78:   required from here
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h:9701:16: warning: cast between incompatible function types from ‘v8::WeakCallbackInfo<node::ObjectWrap>::Callback’ {aka ‘void (*)(const v8::WeakCallbackInfo<node::ObjectWrap>&)’} to ‘Callback’ {aka ‘void (*)(const v8::WeakCallbackInfo<void>&)’} [-Wcast-function-type]
                reinterpret_cast<Callback>(callback), type);
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h: In instantiation of ‘void v8::PersistentBase<T>::SetWeak(P*, typename v8::WeakCallbackInfo<P>::Callback, v8::WeakCallbackType) [with P = Nan::ObjectWrap; T = v8::Object; typename v8::WeakCallbackInfo<P>::Callback = void (*)(const v8::WeakCallbackInfo<Nan::ObjectWrap>&)]’:
../node_modules/nan/nan_object_wrap.h:66:61:   required from here
/home/spaceboyross/.node-gyp/11.0.0/include/node/v8.h:9701:16: warning: cast between incompatible function types from ‘v8::WeakCallbackInfo<Nan::ObjectWrap>::Callback’ {aka ‘void (*)(const v8::WeakCallbackInfo<Nan::ObjectWrap>&)’} to ‘Callback’ {aka ‘void (*)(const v8::WeakCallbackInfo<void>&)’} [-Wcast-function-type]
../src/FontManager.cc: In instantiation of ‘Nan::NAN_METHOD_RETURN_TYPE getAvailableFonts(Nan::NAN_METHOD_ARGS_TYPE) [with bool async = true; Nan::NAN_METHOD_RETURN_TYPE = void; Nan::NAN_METHOD_ARGS_TYPE = const Nan::FunctionCallbackInfo<v8::Value>&]’:
../src/FontManager.cc:213:67:   required from here
../src/FontManager.cc:108:74: warning: cast between incompatible function types from ‘void (*)(uv_work_t*)’ {aka ‘void (*)(uv_work_s*)’} to ‘uv_after_work_cb’ {aka ‘void (*)(uv_work_s*, int)’} [-Wcast-function-type]
     uv_queue_work(uv_default_loop(), &req->work, getAvailableFontsAsync, (uv_after_work_cb) asyncCallback);
                                                                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../src/FontManager.cc: In instantiation of ‘Nan::NAN_METHOD_RETURN_TYPE getAvailableFonts(Nan::NAN_METHOD_ARGS_TYPE) [with bool async = false; Nan::NAN_METHOD_RETURN_TYPE = void; Nan::NAN_METHOD_ARGS_TYPE = const Nan::FunctionCallbackInfo<v8::Value>&]’:
../src/FontManager.cc:214:72:   required from here
../src/FontManager.cc:108:74: warning: cast between incompatible function types from ‘void (*)(uv_work_t*)’ {aka ‘void (*)(uv_work_s*)’} to ‘uv_after_work_cb’ {aka ‘void (*)(uv_work_s*, int)’} [-Wcast-function-type]
../src/FontManager.cc: In instantiation of ‘Nan::NAN_METHOD_RETURN_TYPE findFonts(Nan::NAN_METHOD_ARGS_TYPE) [with bool async = true; Nan::NAN_METHOD_RETURN_TYPE = void; Nan::NAN_METHOD_ARGS_TYPE = const Nan::FunctionCallbackInfo<v8::Value>&]’:
../src/FontManager.cc:215:51:   required from here
../src/FontManager.cc:135:66: warning: cast between incompatible function types from ‘void (*)(uv_work_t*)’ {aka ‘void (*)(uv_work_s*)’} to ‘uv_after_work_cb’ {aka ‘void (*)(uv_work_s*, int)’} [-Wcast-function-type]
     uv_queue_work(uv_default_loop(), &req->work, findFontsAsync, (uv_after_work_cb) asyncCallback);
                                                                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../src/FontManager.cc: In instantiation of ‘Nan::NAN_METHOD_RETURN_TYPE findFonts(Nan::NAN_METHOD_ARGS_TYPE) [with bool async = false; Nan::NAN_METHOD_RETURN_TYPE = void; Nan::NAN_METHOD_ARGS_TYPE = const Nan::FunctionCallbackInfo<v8::Value>&]’:
../src/FontManager.cc:216:56:   required from here
../src/FontManager.cc:135:66: warning: cast between incompatible function types from ‘void (*)(uv_work_t*)’ {aka ‘void (*)(uv_work_s*)’} to ‘uv_after_work_cb’ {aka ‘void (*)(uv_work_s*, int)’} [-Wcast-function-type]
../src/FontManager.cc: In instantiation of ‘Nan::NAN_METHOD_RETURN_TYPE findFont(Nan::NAN_METHOD_ARGS_TYPE) [with bool async = true; Nan::NAN_METHOD_RETURN_TYPE = void; Nan::NAN_METHOD_ARGS_TYPE = const Nan::FunctionCallbackInfo<v8::Value>&]’:
../src/FontManager.cc:217:49:   required from here
../src/FontManager.cc:164:65: warning: cast between incompatible function types from ‘void (*)(uv_work_t*)’ {aka ‘void (*)(uv_work_s*)’} to ‘uv_after_work_cb’ {aka ‘void (*)(uv_work_s*, int)’} [-Wcast-function-type]
     uv_queue_work(uv_default_loop(), &req->work, findFontAsync, (uv_after_work_cb) asyncCallback);
                                                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../src/FontManager.cc: In instantiation of ‘Nan::NAN_METHOD_RETURN_TYPE findFont(Nan::NAN_METHOD_ARGS_TYPE) [with bool async = false; Nan::NAN_METHOD_RETURN_TYPE = void; Nan::NAN_METHOD_ARGS_TYPE = const Nan::FunctionCallbackInfo<v8::Value>&]’:
../src/FontManager.cc:218:54:   required from here
../src/FontManager.cc:164:65: warning: cast between incompatible function types from ‘void (*)(uv_work_t*)’ {aka ‘void (*)(uv_work_s*)’} to ‘uv_after_work_cb’ {aka ‘void (*)(uv_work_s*, int)’} [-Wcast-function-type]
../src/FontManager.cc: In instantiation of ‘Nan::NAN_METHOD_RETURN_TYPE substituteFont(Nan::NAN_METHOD_ARGS_TYPE) [with bool async = true; Nan::NAN_METHOD_RETURN_TYPE = void; Nan::NAN_METHOD_ARGS_TYPE = const Nan::FunctionCallbackInfo<v8::Value>&]’:
../src/FontManager.cc:219:61:   required from here
../src/FontManager.cc:204:71: warning: cast between incompatible function types from ‘void (*)(uv_work_t*)’ {aka ‘void (*)(uv_work_s*)’} to ‘uv_after_work_cb’ {aka ‘void (*)(uv_work_s*, int)’} [-Wcast-function-type]
     uv_queue_work(uv_default_loop(), &req->work, substituteFontAsync, (uv_after_work_cb) asyncCallback);
                                                                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../src/FontManager.cc: In instantiation of ‘Nan::NAN_METHOD_RETURN_TYPE substituteFont(Nan::NAN_METHOD_ARGS_TYPE) [with bool async = false; Nan::NAN_METHOD_RETURN_TYPE = void; Nan::NAN_METHOD_ARGS_TYPE = const Nan::FunctionCallbackInfo<v8::Value>&]’:
../src/FontManager.cc:220:66:   required from here
../src/FontManager.cc:204:71: warning: cast between incompatible function types from ‘void (*)(uv_work_t*)’ {aka ‘void (*)(uv_work_s*)’} to ‘uv_after_work_cb’ {aka ‘void (*)(uv_work_s*, int)’} [-Wcast-function-type]
make: *** [fontmanager.target.mk:102: Release/obj.target/fontmanager/src/FontManager.o] Error 1
make: Leaving directory '/home/spaceboyross/RDE/src/panel/node_modules/font-manager/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/spaceboyross/.nvm/versions/node/v11.0.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:240:12)
gyp ERR! System Linux 4.18.16-arch1-1-ARCH
gyp ERR! command "/home/spaceboyross/.nvm/versions/node/v11.0.0/bin/node" "/home/spaceboyross/.nvm/versions/node/v11.0.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/spaceboyross/RDE/src/panel/node_modules/font-manager
gyp ERR! node -v v11.0.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
npm WARN @ross-technologies/rde-panel@1.0.0 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! font-manager@0.2.2 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the font-manager@0.2.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/spaceboyross/.npm/_logs/2018-11-05T21_15_36_347Z-debug.log
```

--- a/src/FontDescriptor.h
+++ b/src/FontDescriptor.h
@@ -45,7 +45,7 @@ public:
   FontWidth width;
   bool italic;
   bool monospace;
-  
+
   FontDescriptor(Local<Object> obj) {
     path = NULL;
     postscriptName = getString(obj, "postscriptName");
@@ -67,8 +67,8 @@ public:
     italic = false;
     monospace = false;
   }
-  
-  FontDescriptor(const char *path, const char *postscriptName, const char *family, const char *style, 
+
+  FontDescriptor(const char *path, const char *postscriptName, const char *family, const char *style,
                  FontWeight weight, FontWidth width, bool italic, bool monospace) {
     this->path = copyString(path);
     this->postscriptName = copyString(postscriptName);
@@ -90,39 +90,39 @@ public:
     italic = desc->italic;
     monospace = desc->monospace;
   }
-  
+
   ~FontDescriptor() {
     if (path)
       delete path;
-    
+
     if (postscriptName)
       delete postscriptName;
-    
+
     if (family)
       delete family;
-    
+
     if (style)
       delete style;
-    
+
     postscriptName = NULL;
     family = NULL;
     style = NULL;
   }
-  
+
   Local<Object> toJSObject() {
     Nan::EscapableHandleScope scope;
     Local<Object> res = Nan::New<Object>();
-    res->Set(Nan::New<String>("path").ToLocalChecked(), Nan::New<String>(path).ToLocalChecked());
-    res->Set(Nan::New<String>("postscriptName").ToLocalChecked(), Nan::New<String>(postscriptName).ToLocalChecked());
-    res->Set(Nan::New<String>("family").ToLocalChecked(), Nan::New<String>(family).ToLocalChecked());
-    res->Set(Nan::New<String>("style").ToLocalChecked(), Nan::New<String>(style).ToLocalChecked());
-    res->Set(Nan::New<String>("weight").ToLocalChecked(), Nan::New<Number>(weight));
-    res->Set(Nan::New<String>("width").ToLocalChecked(), Nan::New<Number>(width));
-    res->Set(Nan::New<String>("italic").ToLocalChecked(), Nan::New<v8::Boolean>(italic));
-    res->Set(Nan::New<String>("monospace").ToLocalChecked(), Nan::New<v8::Boolean>(monospace));
+    Nan::Set(res, Nan::New<String>("path").ToLocalChecked(), Nan::New<String>(path).ToLocalChecked());
+    Nan::Set(res, Nan::New<String>("postscriptName").ToLocalChecked(), Nan::New<String>(postscriptName).ToLocalChecked());
+    Nan::Set(res, Nan::New<String>("family").ToLocalChecked(), Nan::New<String>(family).ToLocalChecked());
+    Nan::Set(res, Nan::New<String>("style").ToLocalChecked(), Nan::New<String>(style).ToLocalChecked());
+    Nan::Set(res, Nan::New<String>("weight").ToLocalChecked(), Nan::New<Number>(weight));
+    Nan::Set(res, Nan::New<String>("width").ToLocalChecked(), Nan::New<Number>(width));
+    Nan::Set(res, Nan::New<String>("italic").ToLocalChecked(), Nan::New<v8::Boolean>(italic));
+    Nan::Set(res, Nan::New<String>("monospace").ToLocalChecked(), Nan::New<v8::Boolean>(monospace));
     return scope.Escape(res);
   }
-  
+
 private:
   char *copyString(const char *input) {
     if (!input)
@@ -132,37 +132,37 @@ private:
     strcpy(str, input);
     return str;
   }
-  
+
   char *getString(Local<Object> obj, const char *name) {
     Nan::HandleScope scope;
-    Local<Value> value = obj->Get(Nan::New<String>(name).ToLocalChecked());
-    
-    if (value->IsString()) {
-      return copyString(*Nan::Utf8String(value));
+    MaybeLocal<Value> value = Nan::Get(obj, Nan::New<String>(name).ToLocalChecked());
+
+    if (!value.IsEmpty() && value.ToLocalChecked()->IsString()) {
+      return copyString(*Nan::Utf8String(value.ToLocalChecked()));
     }
-  
+
     return NULL;
   }
-  
+
   int getNumber(Local<Object> obj, const char *name) {
     Nan::HandleScope scope;
-    Local<Value> value = obj->Get(Nan::New<String>(name).ToLocalChecked());
-    
-    if (value->IsNumber()) {
-      return value->Int32Value();
+    MaybeLocal<Value> value = Nan::Get(obj, Nan::New<String>(name).ToLocalChecked());
+
+    if (!value.IsEmpty() && value.ToLocalChecked()->IsNumber()) {
+      return value.ToLocalChecked()->Int32Value(Nan::GetCurrentContext()).FromJust();
     }
-    
+
     return 0;
   }
-  
+
   bool getBool(Local<Object> obj, const char *name) {
     Nan::HandleScope scope;
-    Local<Value> value = obj->Get(Nan::New<String>(name).ToLocalChecked());
-    
-    if (value->IsBoolean()) {
-      return value->BooleanValue();
+    MaybeLocal<Value> value = Nan::Get(obj, Nan::New<String>(name).ToLocalChecked());
+
+    if (!value.IsEmpty() && value.ToLocalChecked()->IsBoolean()) {
+      return value.ToLocalChecked()->BooleanValue(Nan::GetCurrentContext()).FromJust();
     }
-    
+
     return false;
   }
 };

--- a/src/FontManager.cc
+++ b/src/FontManager.cc
@@ -20,7 +20,7 @@ Local<Array> collectResults(ResultSet *results) {
 
   int i = 0;
   for (ResultSet::iterator it = results->begin(); it != results->end(); it++) {
-    res->Set(i++, (*it)->toJSObject());
+    Nan::Set(res, i++, (*it)->toJSObject());
   }
 
   delete results;

